src/praisonai/praisonai/agents_generator.py

```
@@ -137,47 +137,15 @@ def sanitize_agent_name_for_autogen_v4(name):
-            overrides = cli_backend_config.get('overrides') or {}
-            return resolve_cli_backend(backend_id, overrides=overrides)
-        raise ValueError(
-            f"cli_backend must be string or dict, got: {type(cli_backend_config).__name__}"
-        )
-    except ImportError:
-        logger.warning(
-            "CLI backend '%s' requested but not available", label
-        )
+        from praisonai.cli_backends import resolve_cli_backend_config
+        return resolve_cli_backend_config(cli_backend_config, logger)
     except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)